### ISSUE TITLE
Do not know how to serialize a BigInt

### ISSUE THREAD

**Issue by morrigan** [getsentry/sentry-electron#237]
## Versions + Platform

- [ ] SDK version - `@sentry/electron@1.3.0`
- [ ] Webpack plugin - `@sentry/webpack-plugin@^1.11.1`
- [ ] Electron version - `electron@8.3.0`
- [ ] Platform - `macOS`

## Description

Seems like BigInt is not supported yet. Wondering if there is a way to sort this out somehow with Sentry and Babel.

```
2020-07-03 12:31:40  []                error: uncaughtException: Do not know how to serialize a BigIntTypeError: Do not know how to serialize a BigInt    at JSON.stringify (<anonymous>)    at eval 

(webpack:///./node_modules/@sentry/electron/dist/renderer/backend.js?:49:81)    at eval 
(webpack:///./node_modules/@sentry/hub/esm/scope.js?:54:21)    at Array.forEach (<anonymous>)    at eval 
(webpack:///./node_modules/@sentry/hub/esm/scope.js?:53:39)    at sentryWrapped 
(webpack:///./node_modules/@sentry/browser/esm/helpers.js?:78:23)
```

**Comment by HazAT** [comment]
@morrigan Are you using master?

**Comment by morrigan** [comment]
> @morrigan Are you using master?

Using 1.3.0 version from NPM

**Comment by timfish** [comment]
We have just released v3.0.0 of the SDK which includes a lot of changes and improvements since this issue was opened. 

If you can still reproduce this with v3, please open a new issue and I'll look into it.
